test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -320,38 +361,46 @@ class test_ucp_sockaddr : public ucp_test {
         }
     }
 
-    void send_recv(entity& from, entity& to, send_recv_type_t send_recv_type,
-                   bool wakeup, ucp_test_base::entity::listen_cb_type_t cb_type)
+    ucs_status_t wait_req_comp(entity &sender, entity &receiver, bool wakeup,
```

Comment:
we already have request_wait in ucp_test base class

Reply:
maybe override progress() somehow?

Reply:
done